webpack.config.js

```diff
@@ -29,6 +28,9 @@ module.exports = {
       }
     ]
   },
+  resolve: {
```

Comment:
thanks for the PR. Is this **resolve** section needed? If so, if possible can you please provide a liner description on why this is needed? I will add this information to README.md.

Reply:
The resolve.extensions is required to avoid errors when you use `import App from './App` to a JSX file. If you dont tell the webpack that you'll use jsx to import it will display an error. 
https://webpack.js.org/configuration/resolve/#resolveextensions